styles/component-styles/image.css

```diff
@@ -0,0 +1,29 @@
+.image-container{
+    display: flex;
+    justify-content: center;
+    border-radius: 2rem;
+    background-color: var(--secondary-container-color);
+    flex-wrap: wrap; 
+    padding: 2rem;
+}
+
+.img-square{
+    object-fit: cover;
+    height: 20rem;
+    width: 20rem;
+}
+
+.img-round{
+    object-fit: cover;
+    height: 20rem;
+    width: 20rem;
+    border-radius: 999px;
+}
+
+.img-responsive{
+    object-fit: cover;
+    width: 50%;
```

Comment:
we can use width 100% so that it will totally depend on the divs width, not on the image.

Reply:
still depends on the div width. Whatever be the div width, image width will be 50% of that.